.github/workflows/teamsync_update_automatically.yml

```diff
@@ -0,0 +1,35 @@
+name: teamSync Update Automatically
+
+on:
+  schedule:
+    - cron: '00 00 * * 1' # UTC time at 00:00 on Monday
+
+jobs:
+  check-permissions:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      with:
+          fetch-depth: 0
+      
+    - name: Set up JDK
+      uses: actions/setup-java@v4
+      with:
+        java-version: '21'
+        distribution: 'adopt'
+        
+    - name: Build project
+      run: mvn clean install
+
+    - name: Update all yml files in permissions directory
```

Comment:
This message appears to be misleading, as it claims to modify yml files, rather than use them as a reference (which seems to be what `YmlTeamLoader#loadTeam` does).